app/helpers/scoring_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module ScoringHelper
+  def number_field_tag_with_buttons(name, value = nil, options = {})
+    if (range = options.delete(:in) || options.delete(:within))
+      options.update(min: range.min, max: range.max)
+    end
+    value ||= options[:value]
+    content_tag 'span', class: 'number-buttons' do
+      concat number_field_tag name, value, options
+      concat increment_button name, value, :-, options
+      concat increment_button name, value, :+, options
+    end
+  end
+
+  def increment_button(target_field, value, type, options)
+    types = { :+ => ['plus', :max], :- => ['minus', :min] }
+    disabled = (options[types[type][1]] && value == options[types[type][1]])
+
+    button_tag types,
+               class: 'btn btn-primary increment',
+               type: :button,
+               data: { field: sanitize_to_id(target_field),
```

Comment:
API dock is down and info on what `sanitize_to_id` is is scarce--what does it do?

Reply:
That gets you e.g. from `onboard_judging[minutes_elapsed]` to `onboard_judging_minutes_elapsed`

That is, from the `name` of the field to the `id` of the field.

Reply:
oh cool